app/models/manageiq/providers/openshift/inventory.rb

```diff
@@ -1,2 +1,39 @@
 class ManageIQ::Providers::Openshift::Inventory < ManageIQ::Providers::Kubernetes::Inventory
+  def self.parser_class_for(ems, target)
+    if !ems.kind_of?(EmsInfra)
+      super
+    else
+      parser_type = if target_is_vm?(target)
+                      "PartialTargetRefresh"
+                    else
+                      "FullRefresh"
+                    end
```

Comment:
Really minor for a follow-up but something this simple taking up 5 lines could be a ternary operator instead on one line
```suggestion
      parser_type = target_is_vm?(target) ? "PartialTargetRefresh" : "FullRefresh"
```

I'm not usually a "hey you can turn this big multi-step thing into one looong line" guy but this is pretty straightforward